api/src/test/java/org/openmrs/api/ConceptServiceTest.java

```diff
@@ -117,6 +153,9 @@ public class ConceptServiceTest extends BaseContextSensitiveTest {
 	@Autowired
 	CacheManager cacheManager;
 
+	@Autowired
+	private MessageSourceService messageSourceService;
```

Comment:
Looks redundant to me!

Reply:
As @mherman22 said, the above is also not needed for this task.